packages/react-ui-core/src/Button/ApplyButton.js

```diff
@@ -12,15 +12,15 @@ import Button from './Button'
 export default class ApplyButton extends PureComponent {
   static propTypes = {
     className: PropTypes.string,
+    children: PropTypes.string,
```

Comment:
Since this is a react thing, shouldn't it be `node`?

Reply:
done && this.done